src/handlers/major_change.rs

```diff
@@ -294,6 +294,10 @@ async fn handle(
             .post_comment(&ctx.github, &comment)
             .await
             .context("post major change comment")?;
+        issue
+            .lock(&ctx.github, None)
+            .await
+            .context("lock major change issue")?;
```

Comment:
```suggestion
            .context("lock Major Change Proposal")?;
```

(I believe it's the official branding :upside_down_face: )